low-level/framework/utils/utility.py

```diff
@@ -166,6 +167,19 @@ def get_current_time():
         """Returns the time as integer number in seconds since the epoch in UTC."""
         return int(time.time())
 
+    @staticmethod
+    def delete_file_and_dir(path, del_dir=False):
```

Comment:
Use function from #480, extend if needed

Reply:
Done, added that function here in common place, extended as per need and using it in reset and cleanup interface.